webservices/common/models/dates.py

```diff
@@ -30,6 +34,10 @@ def report_type_full(self):
         return clean_report_type(self.report.report_type_full)
 
 
+class ReportDateDisclosure(DisclosureMixin, ReportDate):
+    pass
```

Comment:
That's the thing, the subset.dump does include the disclosure schema, but I was getting failures in `test_dates` specifically the test `test_reporting_date_filters` at https://github.com/18F/openFEC/blob/develop/tests/test_dates.py#L41.  This test hits the one model where the disclosure model is hit directly.  And was causing failures, stating that it couldn't find the table that backs that model (one of the tables that was migrated to the disclosure schema).  As far as I can tell the reason being is that the methodology the tests use don't recognize the syntax that points a model to a different schema.  Hence that's why I used a mixin.